eacl/table.go

```diff
@@ -199,3 +200,60 @@ func (t *Table) UnmarshalJSON(data []byte) error {
 
 	return nil
 }
+
+// EqualTo compares Table between each other.
```

Comment:
```suggestion
// EqualsTo compares Table between each other.
```
or
```suggestion
// IsEqualTo compares Table between each other.
```
or just
```suggestion
// Equals compares Table between each other.
```
(pardon my english if i'm wrong)

Reply:
Let's also make it clear in the docs that the operation is sensitive to the order of the list data.

Reply:
> EqualsTo

if it is a verb, it should not be used with "to", so "EqualTo"/"IsEqualTo" or "Equals"